components/brave_shields/browser/ad_block_engine.cc

```diff
@@ -157,8 +158,7 @@ void AdBlockEngine::EnableTag(const std::string& tag, bool enabled) {
     }
   } else {
     ad_block_client_->removeTag(tag);
-    std::set<std::string>::iterator it =
-        std::find(tags_.begin(), tags_.end(), tag);
+    std::set<std::string>::iterator it = base::ranges::find(tags_, tag);
```

Comment:
i guess this is simply `tags_.erase(tag);`

Reply:
Fixed.